### PR TITLE
Change getQueryResult to avoid using API queries

### PR DESCRIPTION
**Pull request by matthew-a-thompson**
see #8

### CODE REVIEW

File: src/ScribuntoLuaLibrary.php [hunk 1]
```diff
@@ -37,29 +37,30 @@ public function register() {
 	 *
 	 * @return array
 	 */
-	public function getQueryResult( $queryString = null ) {
```
Comment (mwjames) [inline src/ScribuntoLuaLibrary.php]: At the top (after the namespace) you need to add something like `use SMWQueryProcessor as QueryProcessor`;

```
..S....SPHP Fatal error:  Class 'SMW\Scribunto\QueryProcessor' not found in /home/travis/build/SemanticMediaWiki/mw/extensions/SemanticScribunto/src/ScribuntoLuaLibrary.php on line 44

PHP Stack trace:

```

**Comment by mwjames** [comment]
I also suspect that tests will fail due to a change in the return structure.

**Comment by matthew-a-thompson** [comment]
I've updated the meta count test to reflect the new return structure. The empty query test will currently fail, as a table is returned that still contains "results" and "printrequests" as keys. Both of these keys point to values of empty tables, so checking that ret.results and ret.printrequests are both empty tables seems like an acceptable test.

**Comment by mwjames** [comment]
@matthew-a-thompson Tests pass :+1: .

@cicalese Can you confirm that this works as expected (before I press the merge button).

**Comment by cicalese** [comment]
Thank you both! We tested the patch and it works great, so you can go ahead and merge.

**Comment by mwjames** [comment]
Thanks for the feedback.

**Pull request merged**